internal/alertmanager/currentalert/process.go

```diff
@@ -93,17 +95,15 @@ func (e *examiner) processAlert(ds disableScaling) error {
 		}
 	case "downScale":
 		if ds.Down {
-			e.Log.Info("downscale is skipped due to minimum broker count")
-			return nil
+			return errors.New("downscaling is skipped due to downscale limit")
 		}
 		err := downScale(e.Alert.Labels, e.Client)
 		if err != nil {
 			return err
 		}
 	case "upScale":
 		if ds.Up {
-			e.Log.Info("upscale is skipped due to maximum broker count")
-			return nil
+			return errors.New("upscaling is skipped due to upscale limit")
```

Comment:
why do we return with error instead of just simply logging the fact the upscale is skipped? Not performing upscale when it's disabled it's not an error.

Reply:
https://github.com/banzaicloud/kafka-operator/pull/243#discussion_r364087820